### PR TITLE
Fix r6 refs

### PR DESCRIPTION
**Pull request by jennybc**
Completing some ref changes started in b12fd37 re: the chapter on R6. Also fixing one typo.

### CODE REVIEW

File: Introduction.Rmd [hunk 1]
```diff
@@ -24,7 +24,7 @@ If you are new to R, you might wonder what makes learning such a quirky language
   for women and other gender minorities.
   
 * A massive set of packages for statistical modelling, machine learning,
-  visualisation, and importing and manipulating data. Whatever model or
+  visualization, and importing and manipulating data. Whatever model or
```
Comment (hadley) [inline Introduction.Rmd]: I use British spellings

Comment (jennybc) [reply Introduction.Rmd]: OK I will fix

**Comment by jennybc** [comment]
Happy now?

**Comment by hadley** [comment]
Looks great! 🇳🇿

**Pull request merged**